content-src/components/Base/Base.jsx

```diff
@@ -152,7 +152,7 @@ export class BaseContent extends React.PureComponent {
             <ConfirmDialog />
           </main>
         </div>
-        {this.props.isFirstrun && <StartupOverlay />}
+        {this.props.isFirstrun && <StartupOverlay fxa_endpoint={prefs["fxaccounts.endpoint"]} />}
```

Comment:
you don't need this prop here, you're getting the prop from the connected state in StartupOverlay.jsx